### DISCUSSION TITLE
Collection share to internet

### DISCUSSION THREAD

**Discussion by RickCogley** [outline/outline#1780] [Ideas]
At this time, it appears that only a single page can be shared to the internet. 

A useful enhancement would be to be able to share an entire collection in this way. You share the collection itself, not just the page. 

In terms of UX, when sharing, the UI would tell the sharing user that they are about to share a whole bunch of pages. Also, users could be prevented from sharing a collection if a member page is private, or, Outline could simply not share that one page.

**Comment by tommoor** [comment]
Please see this issue: https://github.com/outline/outline/issues/1160
